src/views/examples/Register.js

```diff
@@ -1,166 +1,181 @@
               <FormGroup>
-                <InputGroup className="input-group-alternative">
+                <InputGroup className="input-group-alternative-2" id="ConfirmPassword">
                   <InputGroupAddon addonType="prepend">
                     <InputGroupText>
-                      <i className="ni ni-lock-circle-open" />
+                      <i className="ni ni-check-bold" />
                     </InputGroupText>
                   </InputGroupAddon>
                   <Input
-                    placeholder="Password"
+                    
+                    onChange={(e) => setConfirmpassword(e.target.value)}
+                    placeholder="Confirm Your Password"
                     type="password"
-                    autoComplete="new-password"
+                    autoComplete="off"
+                    onKeyUp={()=> checkPassword(password,confirmpassword)}
+                    
                   />
                 </InputGroup>
               </FormGroup>
-              <div className="text-muted font-italic">
-                <small>
-                  password strength:{" "}
-                  <span className="text-success font-weight-700">strong</span>
-                </small>
-              </div>
-              <Row className="my-4">
-                <Col xs="12">
-                  <div className="custom-control custom-control-alternative custom-checkbox">
-                    <input
-                      className="custom-control-input"
-                      id="customCheckRegister"
-                      type="checkbox"
-                    />
-                    <label
-                      className="custom-control-label"
-                      htmlFor="customCheckRegister"
-                    >
-                      <span className="text-muted">
-                        I agree with the{" "}
-                        <a href="#pablo" onClick={(e) => e.preventDefault()}>
-                          Privacy Policy
-                        </a>
-                      </span>
-                    </label>
-                  </div>
-                </Col>
-              </Row>
+              <FormGroup>
+                <div className="text-muted text-center mt-2 mb-4">
+                  <span style={{
+                      fontWeight: 'light',
+                      color: 'red',
+                      alignItems: 'right'
+                      }}>{pass}
+                  </span>
+                </div>
+              </FormGroup>
               <div className="text-center">
-                <Button className="mt-4" color="primary" type="button">
-                  Create account
+                <Button className="mt-4" 
+                        id="submit"
+                        disabled={isDisabled}
+                        color="primary" 
+                        type="submit"
+                        onClick={setEmailError==="valid email"? (e)=> onRegister(e) : undefined}
```

Comment:
Use this condition inside onRegister functions. Just use this onClick={(e)=> onRegister(e)} and check conditions at start of function and if false return from that function. Or you can have some disable button for this as well.